components/Grid/grid.css

```diff
@@ -0,0 +1,95 @@
+.grid_container {
+  margin: 1rem 0.5rem;
+}
+
+.grid_main_container {
+  padding: 2rem;
+  margin: 1rem;
+  border: 1px solid lightgrey;
+  display: grid;
+  border-radius: 0.5rem;
+}
+
+.grid_50_50 {
+  grid-template-columns: 50% 50%;
+}
+
+.grid_70_30 {
+  grid-template-columns: 70% 30%;
+}
+
+.grid_30_70 {
+  grid-template-columns: 30% 70%;
+}
+
+.img_responsive {
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.grid_content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
+  text-align: center;
+  padding: 0 2rem;
+}
+
+.grid_content button {
+  padding: 0.5rem;
+  font-size: 1.1rem;
+  border: 1px solid lightgrey;
```

Comment:
same as above, lightgrey ? if using color name across your code base then pls ignore it